www/front_src/src/Resources/Details/tabs/Details/DetailsCard/Groups.tsx

```diff
@@ -1,46 +1,48 @@
+/* eslint-disable react/jsx-key */
```

Comment:
```suggestion
```

Reply:
i deleted this before pushing , command  eslint:fix can re-add this rules ?